### PR TITLE
feat(lpa-questionnaire-web-app): created view

### PR DESCRIPTION
**Pull request by swdpcomputing**
## Ticket Number

https://pins-ds.atlassian.net/browse/AS-1851

## Description of change

created view

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/lpa-questionnaire-web-app/src/routes/questionnaire-submitted.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const questionnaireSubmittedController = require('../controllers/questionnaire-submitted');
+const fetchExistingAppealReplyMiddleware = require('../middleware/fetch-existing-appeal-reply');
+const fetchAppealMiddleware = require('../middleware/fetch-appeal');
+
+const router = express.Router();
+
+router.get(
+  '/:id/questionnaire-submitted',
+  // Currently unknown where email address will come from, so keeping both middleware
+  [fetchAppealMiddleware, fetchExistingAppealReplyMiddleware],
```
Comment (davehlewis) [inline packages/lpa-questionnaire-web-app/src/routes/questionnaire-submitted.js:11]: Are these middlewares needed here? Could save a calll to the APIs/session

**Review by davehlewis — CHANGES_REQUESTED** [review]

File: packages/lpa-questionnaire-web-app/src/routes/questionnaire-submitted.js (hunk 1 above)
Comment (swdpcomputing) [reply packages/lpa-questionnaire-web-app/src/routes/questionnaire-submitted.js:11]: Was leaving in awaiting email address task. Suppose not necessary yet.

Comment (davehlewis) [reply packages/lpa-questionnaire-web-app/src/routes/questionnaire-submitted.js:11]: makes sense. In future will only need one of these anyway

**Review by davehlewis — APPROVED** [review]
LGTM

**Pull request merged**